volto/override-components.rst

```diff
@@ -0,0 +1,81 @@
+.. _override_components-label:
+
+===================
+Override Components
+===================
+
+Override The Logo
+=================
+
+When we want to override a specific file we can create an alias pointing to
+our own theme. So for example if we want to replace the logo, which is located
+in Volto at :file:`components/theme/Logo/Logo.svg`, we will add an logo to
+our theme and create an alias. The folder structure needs to match the folder
+structure of Volto in the :file:`customizations` folder.
+
+Excercise
+=========
+
+Replace the logo with a hello kitty logo.
```

Comment:
Warning! Copyright infringement! Suggest not using something fiercely defended by attorneys.